app/controllers/location_reports_controller.rb

```diff
@@ -0,0 +1,52 @@
+# This file is part of SEQUENCESCAPE; it is distributed under the terms of
+# GNU General Public License version 1 or later;
+# Please refer to the LICENSE and README files for information on licensing and
+# authorship of this file.
+# Copyright (C) 2018 Genome Research Ltd.
+
+class LocationReportsController < ApplicationController
+  before_action :login_required
+
+  def index
+    @studies                = Study.alphabetical.pluck(:name, :id)
+    @plate_purposes         = PlatePurpose.pluck(:name, :id).sort
```

Comment:
Add include SharedBehaviour::Named to PlatePurpose, and use alphabetical scope instead.